### PR TITLE
readme change and added another example

### PR DESCRIPTION
**Pull request by koaning**

### CODE REVIEW

**Comment by koaning** [comment]
Because we're using the free version of travis, we sometimes need to remind travis to run a test twice because sometimes it does not have sufficient disk available. My guess is that this occurs when we are running two tests on travis at the same time. Dunno for sure.

File: .travis.yml [hunk 1]
```diff
@@ -5,5 +5,6 @@ script:
   - pip install pytest
   - python setup.py install
   - pytest
+  - python examples/readme_example.py
```
Comment (rogiervandergeer) [inline .travis.yml]: I like the example! It's clear and I hope easy to follow. I do think we need to call the thing by it's name; e.g. something like `optimise_nonlinear`.

File: README.md [hunk 2]
```diff
@@ -48,41 +48,83 @@ One could even combine the two algorithms into a new one:
...
-    def pick_n_parents(population, num_parents):
-        return [random.choice(population) for i in range(num_parents)]
+    evo2 = (Evolution()
+           .survive(n=1)
+           .mutate(func=add_noise, sigma=0.2)
+           .breed(parent_picker=pick_random_parents, combiner=make_child))
 
-    def crossover(*parents):
-        ...
+    evo3 = (Evolution()
```
Comment (rogiervandergeer) [inline README.md]: I think we should mention that of course you're not obliged to make two or three evolutions.

File: .travis.yml (hunk 1 above)
Comment (koaning) [reply .travis.yml]: I've called it `simple_nonlinear` for now.

File: README.md (hunk 2 above)
Comment (koaning) [reply README.md]: I'll add that as comments.

**Pull request merged**